sdk/personalizer/Azure.AI.Personalizer/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 2.0.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 2.0.0-beta.2 (2022-03-08)
 
```

Comment:
@johnhuang01, you can add updates here under sections such as -
```
### Features Added

### Breaking Changes

### Bugs Fixed

### Other Changes
```

Reply:
Features added:

- Provide customers with much lower latency and more transactions per second than the current service limits.
- Provide subsampling for customers so that only subsampled data are processed.
- Provide a model import API to allow customers to warm start the Personalizer service.

Breaking changes:

- In PersonalizerAdministrationClient, GetPersonalizerModel and GetPersonalizerModelAsync are replaced with ExportPersonalizerModel and ExportPersonalizerModelAsync.
- In PersonalizerClientOptions, ServiceVersion is changed from V1_1_preview_1 to V1_1_preview_3.